.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,27 @@
+name: dsp-appsec-trivy
+on:
+  pull_request:
+  schedule:
+    - cron:  '0 14 * * 1' # each Monday at 9am EST
+
+jobs:
+  appsec-trivy:
+    # Parse Dockerfile and build, scan image if a "blessed" base image is not used
+    name: DSP AppSec Trivy check
+    runs-on: ubuntu-latest
+    env:
+      SARIF_PATH: trivy-results.sarif
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: broadinstitute/dsp-appsec-trivy-action@v1
+        with:
+          image: broadinstitute/firecloud-ui:develop
```

Comment:
Sorry to nitpick, but if we're using the `develop` image as the scanning target, is it going to build the PR changes?

Reply:
In this example it's just going to scan the existing image. It'd be better to rebuild it instead though - I'll add that.

Reply:
OK, now I remember the context. It looks like the base image in this case is private, so we can't really rebuild it. So I've just reverted the changes to only scan `develop` on schedule.